apps/website/src/pages/docs/stepper.mdx

```diff
@@ -9,19 +9,46 @@ group: core
 
 import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
-import Placeholder from '~/components/Placeholder.astro';
 import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-stepper' />
-
 <LiveExample src='Stepper.main.tsx'>
   <AllExamples.StepperMainExample client:load />
 </LiveExample>
 
 In defined, multi-step user interactions, it is useful to inform the user of the number of steps involved in the process, and the current step within that sequence. A UI 'Stepper' guides the user through the steps and provides some interaction to return to previous steps. A standard example of a Stepper would be an e-Commerce checkout process, but there are many others where the user must perform sequential steps.
 
+## Variants
+
+There are two different styles of stepper, depending on the amount of steps required to complete the process.
+
+### Short Stepper
+
+The default "short" stepper example is suggested for processes with four (4) steps or less. Each step is named under the circle. This stepper is suggested if the step names can be summarized with very short labels. For example, instead of “Complete Billing Information”, simply use “Billing.”
+
+The user may click on previous steps to go back. [Tooltips](tooltip) can also be implemented to give more information about a particular step.
+
+Designers should confirm the display is acceptable due to the number of steps and labels at various resolutions. If text wrapping occurs or the stepper's visual quality decreases significantly when the window is re-sized, use the long stepper instead (see below).
```

Comment:
```suggestion
Designers should confirm the display is acceptable due to the number of steps and labels at various resolutions. If text wrapping occurs or the stepper's visual quality decreases significantly when the window is re-sized, use the [long stepper](#long-stepper) instead.
```
We can link to that section instead of saying see below.

Reply:
Fixed